linting/package.json

```diff
@@ -12,6 +12,7 @@
   "devDependencies": {
     "babel-eslint": "^10.0.3",
     "eslint": "^6.7.2",
-    "eslint-plugin-ember": "kategengler/eslint-plugin-ember#065cacd5b7b6fc5378f22b02fe49c53e10392d89"
+    "eslint-plugin-ember": "^10.5.0",
+    "eslint-plugin-ember-observer": "mansona/eslint-plugin-ember-observer#631a7d3b3ce5f1419a68c522ac3933c88be879bf"
```

Comment:
Will merge once this updated. Since we have our own plugin now we should use tags.

Reply:
I've updated the repo link now and I've moved over to using tags 👍